board/management/commands/getgamefromapi.py

```diff
@@ -38,33 +38,25 @@ def handle(self, *args, **options):
             for i in range(len(game.screen_url)):
                 Image.objects.update_or_create(url=game.screen_url[i], defaults={'game': g1})
 
-            platforms = Platforms.objects.filter(name__in=game.platforms)
+            platforms = Platform.objects.filter(name__in=game.platforms)
             genres = Genre.objects.filter(name__in=game.genres)
-            for platform in platforms:
-                g1.platforms.add(platform)
-            for genre in genres:
-                g1.genres.add(genre)
+            g1.platforms.add(*platforms)
```

Comment:
`g1` is a bad variable name, you can call it `game` since it is a `Game` instance and the instance of `GameAPI` can be called `game_api_response_object` or smth like that.